src/benchInsert.c

```diff
@@ -1530,6 +1538,7 @@ static int startMultiThreadInsertData(int db_index, int stb_index) {
                         stbInfo->lenOfCols * g_arguments->reqPerReq + 1024;
                 }
                 pThreadInfo->buffer = calloc(1, pThreadInfo->max_sql_len);
+                g_memoryUsage += pThreadInfo->max_sql_len;
```

Comment:
max_sql_len is always MAX_SQL_LEN now?

Reply:
for restful and taosc insert, is always MAX_SQL_LEN, for stmt and schemaless, especially for influxdb line and telnet protocol, which need string buffer to insert, will not have the MAX_SQL_LEN limitation, instead it only depend on the batch size, so pThreadInfo->max_sql_len for those insertion is dynamic